### PR TITLE
ignore compliance tests for component readiness for now

### PR DESCRIPTION
**Pull request by xueqzhan**

### CODE REVIEW

File: pkg/api/component_report.go [hunk 1]
```diff
@@ -28,7 +28,7 @@ import (
 const (
 	triagedIncidentsTableID = "triaged_incidents"
 
-	ignoredJobsRegexp = `-okd|-recovery|aggregator-|alibaba|-disruptive|-rollback|-out-of-change|-sno-fips-recert`
+	ignoredJobsRegexp = `-okd|-recovery|aggregator-|alibaba|-disruptive|-rollback|-out-of-change|-sno-fips-recert|periodic-ci-ComplianceAsCode-|efs|interop`
```
Comment (dgoodwin) [inline pkg/api/component_report.go:31]: Actually ken that efs string looks dangerous, could that not pick up a lot of unintended jobs?

Comment (dgoodwin) [reply pkg/api/component_report.go:31]: Only one hit in sippy: pull-ci-openshift-aws-efs-csi-driver-operator-master-operator-e2e-extended
Would prefer -efs- though just to be safe, maybe same for -interop-.

Comment (xueqzhan) [reply pkg/api/component_report.go]: Done

**Pull request closed without merging**